src/apps/rDSN.monitor/app_package/static/js/analyzer.js

```diff
@@ -0,0 +1,420 @@
+$(function () {
+    $('.list-group.checked-list-box .list-group-item').each(function () {
+        
+        // Settings
+        var $widget = $(this),
+            $checkbox = $('<input type="checkbox" class="hidden" />'),
```

Comment:
use more meaningful var name - cannot understand what it is for using 'checkbox'. similar cases like widget or the others should also be fixed